src/apps/bridge/sensor_drivers/rbrCodaSensor.h

```diff
@@ -21,7 +21,7 @@ typedef struct rbr_coda_aggregations_s {
 typedef struct RbrCodaSensor : public AbstractSensor {
   uint32_t rbr_coda_agg_period_ms;
   AveragingSampler temp_deg_c;
-  AveragingSampler pressure_ubar;
+  AveragingSampler pressure_deci_bar;
```

Comment:
The values stored in this buffer are actually the dbar values that come directly from the RBR sensor.